src/flamenco/runtime/tests/fd_exec_sol_compat.c

```diff
@@ -89,6 +89,7 @@ sol_compat_instr_execute_v1( uchar *       out,
                              ulong         in_sz ) {
 
   ulong fmem[ 64 ];
+  memset( smem, 0, smax );
```

Comment:
This is a hack, if this solves your problem it means there is likely some bad memory access further on. However, for now I would say it's fine to keep it in `sol_compat_instr_execute_v1`, with a FIXME comment to address it later.

Reply:
Does this also significantly slow down test execution?